examples/google-tables/helpers.ts

```diff
@@ -0,0 +1,141 @@
+import type {Column} from "./types";
+import type {Table} from "./types";
+import * as coda from "@codahq/packs-sdk";
+import { getConverter } from "./convert";
+
+const BaseUrl = "https://area120tables.googleapis.com/v1alpha1";
+const PageSize = 100;
+const ShortCacheTimeSecs = 60;
+
+export function getTableUrl(tableName: string): string {
+  return coda.joinUrl(BaseUrl, tableName);
+}
+
+// Get the available tables from the API.
+export async function getTables(context: coda.ExecutionContext)
+  : Promise<Table[]> {
+  let url = coda.withQueryParams(coda.joinUrl(BaseUrl, "tables"), {
```

Comment:
`const` for the variable declarations throughout.

Reply:
Ditto to above.